### ISSUE TITLE
Graphical selection of a path point does not highlight the Component (e.g. Muscle) that in the Navigator

### ISSUE THREAD

**Issue by aseth1** [opensim-org/opensim-gui#814]
### Steps to reproduce
Load arm26.osim. Select any path point of a muscle. 

### Expected result
In 3.3 the muscle is highlighted in the Navigator. This is super convenient for identify muscles visually.
![image](https://user-images.githubusercontent.com/6494122/40812446-c6303022-64ea-11e8-8977-707a562e51b2.png)

### Actual result
Any previous selected object remains highlighted in the Navigator. Selected muscle is not indicated.
![image](https://user-images.githubusercontent.com/6494122/40812426-a65e6552-64ea-11e8-9a11-5054c295c007.png)

### Environment and GUI version
Windows OpenSim-43c1be65-2018-05-30

**Comment by aseth1** [comment]
This is particularly useful for models with many muscles and you want to identify the muscle quickly by its path in the Visualizer.

**Comment by aymanhab** [comment]
Agreed @aseth1 For the time being you should see the name of the selected path point on the status bar.

**Comment by aseth1** [comment]
>For the time being you should see the name of the selected path point on the status bar.

Cool! Who knew we had a status bar in the bottom left corner?

**Comment by tkuchida** [comment]
> Who knew we had a status bar in the bottom left corner?

:raising_hand_man: but only because of #542

**Comment by aseth1** [comment]
In Windows OpenSim_9c841136-2018-06-27 I am unable to select muscles graphically.

In the previous case, I still don't think it is correct for the highlighted item in the navigator to be unrelated to the selected item in the Visualizer and to then rely on the status bar to decipher which muscle the path point belongs too. How is a user supposed to know which information is reliable? The cognitive load on the user is going up not down.
